src/js/components/aboutTheData/TimeFilters.jsx

```diff
@@ -118,7 +118,7 @@ const TimePeriodFilters = ({
     }, [submissionPeriods, urlFy, urlPeriod, latestPeriod, latestFy]);
 
     const generatePeriodDropdown = (fy, periods) => (
-        parsePeriods(fy, periods)
+        parsePeriods(fy, periods.toJS())
```

Comment:
The FYs are still showing up in ascending order for me.

Reply:
[Gonna put this in pt II](https://github.com/fedspendingtransparency/usaspending-website/commit/6ee7e903aed24db7f6e78c4a487020753f6e38a8)